README.md

```diff
@@ -1,18 +1,680 @@
+
+For example:
+
+```json
+{
+  "file_name": "foundations.md",
+  "id": "foundations",
+  "file_dir": "application-development/golang",
+  "title": "Go SDK developer's guide - Foundations",
+  "sidebar_label": "Foundations", // This is the label of the guide in the sidebar
+  "sidebar_position": 1, // This is the position within the sidebar relative to others at the same level
+  "description": "The Foundations section of the Temporal Go SDK Developer's guide covers the minimum set of concepts and implementation details needed to build and run a Temporal Application in Go – that is, all the relevant steps to start a Workflow Execution that executes an Activity.",
+  "use_description": false,
+  "toc_max_heading_level": 4,
+  "add_tabs_support": true,
+  "sections": [
+    {
+      "type": "p", // use p for sections that you don't want a header for or want to string together.
+      "id": "go/foundations"
+    },
+    {
+      "type": "h2",
+      "id": "go/add-sdk"
+    },
+    {
+      "type": "h2", // this is the header level you want
+      "id": "how-to-develop-a-workflow-in-go" // this is the id of the info node
+    }
+    // ...
+}
+```
+
+#### Run Assembly one more time
+
+In the documentation repo, run `yarn assemble` to generate the guides from the guide configurations.
+
+Locally you can preview your work in the docs repo using `yarn start`
+
+### How to use Snipsync
+
+Make sure your source code repository is registered in the `snipsync.config.yaml` file.
+
+In the `docs-src` Information Nodes, place your [snipsync](https://github.com/temporalio/snipsync) snippet wrappers where you want the snippet to be.
+
+```
+<!--SNIPSTART typescript-hello-client -->
+<!--SNIPEND-->
+
+```
+
+Run `yarn snipsync`, to merge the snippet contents (in this case, from `[samples-typescript](https://github.com/temporalio/samples-typescript/blob/75bdcd613bd24f8f357cb96d1b83051353c5685a/hello-world/src/client.ts#L1)`) between the `SNIPSTART` and `SNIPEND` tags.
+
+[Run the Assembly Workflow](https://www.notion.so/Documentation-repo-README-content-WIP-ddd0e586594246b3b9948919bbf7b12b?pvs=21).
+
+Format the files with `yarn format` .
+
+[Put up a Pull Request](#how-to-put-up-a-pull-request).
+
+### How to put up a Pull Request
+
+Commit your changes to a new branch.
+
+If you are outside the Temporal organization, you must fork the repository and create Pull Requests from branches on your own fork.
+
+The README in GitHub's [first-contributions repo](https://github.com/firstcontributions/first-contributions) provides an example.
+
+You get a deployment preview for each push to the Pull Request.
+
+!https://github.com/temporalio/documentation/raw/main/static/img/readme/vercel-deploy-preview.png
+
+If you don't belong to the Temporal organization, a member of the Temporal @temporalio/education team will need to approve your preview.
+
+If you don't belong to the Temporal organization, you may need to sign the Contributor License Agreement yourself.
```

Comment:
So, Temporal organization or temporalio? You used temporalio earlier. Wanted to give you a choice for what to use from here on out.